### PR TITLE
Add guide of how to use confidence thresholds with the Vision framework.

### PR DESCRIPTION
**Pull request by syoutsey**
Fixes https://github.com/apple/turicreate/issues/798.

### CODE REVIEW

File: userguide/object_detection/export-coreml.md [hunk 1]
```diff
@@ -103,11 +103,37 @@ Before we discuss how to use this threshold, we must first make a prediction.
 
 ##### Prediction
 
-Making a prediction is easy:
+Making a prediction is easy. Starting in iOS 13 we can now include confidence thresholds
+directly in the `VNCoreMLModel`. For an example of how to include thresholds in a project utiziling the Vision framework,
```
Comment (shantanuchhabra) [inline userguide/object_detection/export-coreml.md]: utiziling -> utilizing

File: userguide/object_detection/export-coreml.md [hunk 2]
```diff
@@ -103,11 +103,37 @@ Before we discuss how to use this threshold, we must first make a prediction.
...
+see [here](https://developer.apple.com/documentation/coreml/understanding_a_dice_roll_with_vision_and_object_detection).
+First, we need to setup a subclass of [`MLFeatureProvider`](https://developer.apple.com/documentation/coreml/mlfeatureprovider):
```
Comment (shantanuchhabra) [inline userguide/object_detection/export-coreml.md]: setup -> set up?

Comment (syoutsey) [reply userguide/object_detection/export-coreml.md]: `Setup` is common and used [elsewhere](https://apple.github.io/turicreate/docs/userguide/drawing_classifier/data-preparation.html) in our UG.

File: userguide/object_detection/export-coreml.md (hunk 1 above)
Comment (syoutsey) [reply userguide/object_detection/export-coreml.md]: Good catch.

**Pull request merged**